e2e/libs/kubelet/kubelet.py

```diff
@@ -0,0 +1,22 @@
+from utility.utility import logging
+from node_exec import NodeExec
+import time
+
+def restart_kubelet(node_name, stop_time_in_sec=10):
+    filepath = "./templates/kubelet/kubelet_restart.sh"
+    dst_filepath = "/tmp/kubelet_restart.sh"
+    with open(filepath, 'r') as f:
+        content = f.read()
+        cmd = [
+            "sh",
+            "-c",
+            f"touch {dst_filepath}; chmod +x {dst_filepath}; cat > {dst_filepath} <<-'EOF'\n{content}\nEOF"
```

Comment:
This looks like it will add a file to the host environment.
Why not just have 2 functions `stop_kubelet` and `start_kubelet` with `systemctl stop/start k3s-agent`?

Reply:
This is feasible after `ssh` implemented.

Reply:
Is it possible with `nsenter`?

Reply:
Never mind, I see your point.

Reply:
I've tried `nsenter` before, but so far unable to set the options/parameters right to make it work. Do you have any suggestion?

Reply:
What about we do something similar to the Litmus helper pod?
https://github.com/litmuschaos/litmus-go/blob/bdddd0d8036cbd57880f64db39817906d69ff823/chaoslib/litmus/kubelet-service-kill/lib/kubelet-service-kill.go#L115

Reply:
It doesn't seem to have much difference?
The command used in Litmus is:
```
sleep 10 && systemctl stop kubelet && sleep " + strconv.Itoa(experimentsDetails.ChaosDuration) + " && systemctl start kubelet
```

Reply:
It doesn't directly create a file on the host. And, we should also consider leverage on `IMAGE_LITMUX = 'litmuschaos/go-runner:latest'`,  which should come equipped with all the necessary packages.

Reply:
More investigation needed. 

Probably related to what's the approach we'd like to execute commands: `Using a permanent pod to run all commands` or `Using a disposable pod to run every command`.

Reply:
We could also enhance later, considering that this PR appears to be fulfilling its purpose. WDYU @yangchiu ?

Reply:
Node execution pod is a delegate permanently existing on each node to execute any random commands for users.

Helper pod is created to execute pre-defined specific commands in its manifest and destroyed once it finished its job.

Yes, it can be refined later since at this moment there's no way to tell which approach is more suitable for which situation.

Reply:
I've confirmed this can be done by a helper pod, but the image is still needed to be [`ubuntu:16.04`](https://github.com/litmuschaos/litmus-go/blob/bdddd0d8036cbd57880f64db39817906d69ff823/pkg/generic/kubelet-service-kill/environment/environment.go), and some configuration need to be done. I can refine in this PR.

Reply:
Refined.